tests/kernel/install_ltp.pm

```diff
@@ -96,6 +96,12 @@ sub run {
     my $self     = shift;
     my $inst_ltp = get_var 'INSTALL_LTP';
     $self->wait_boot;
+
+    # poo#18980
+    if (check_var('ARCH', 'ppc64le')) {
+        add_serial_console('hvc1');
+    }
```

Comment:
Ah, maybe we could do this in test/shutdown/shutdown.pm as well? A grep of the code base didn't reveal any other uses of `hvc1` so it shouldn't effect any other tests. Then @jtzhao's code can start using virtio console on ppc64le with no modification.